contracts/Core/AssetManager.sol

```diff
@@ -73,29 +75,31 @@ contract AssetManager is ACL, AssetStorage, Constants {
         bool repeat,
         string calldata name,
         string calldata selector,
-        string calldata url
+        string calldata url,
+        int8 power
```

Comment:
same

Reply:
sure will move it up

Reply:
done https://github.com/razor-network/contracts/pull/330/commits/e84c66c07831a6dbd62038d21726d9d65899a153